cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    pk = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=133)
+    description = serializers.CharField()
```

Comment:
it's better to add max_length here as well.
or u can make this field in model TextField and after make tge same in serializer)